src/components/Image/Image.test.tsx

```diff
@@ -33,34 +41,22 @@ describe("Image", () => {
     "QpLjFbruJsPcGSCp6ET6DCrNQeWFsRVaM2Co99ewZjLuY42kdpBEXjcw9HPcTjKKZw141sK" +
     "BNOoFfNMueYaHtNjNI";
 
-  it("does not render an image src until it is 'inView'", () => {
+  it("lazy loading does not render an image src until it is 'inView'", async () => {
```

Comment:
For 2, I'm not sure how to test this because the previous test which was working is no longer working. So I'll have to update this at a later time. It works, just hard to test.